apps/gitness/src/components-v2/breadcrumbs/breadcrumbs.tsx

```diff
@@ -12,32 +12,24 @@ function Breadcrumbs() {
       <Topbar.Left>
         <Breadcrumb.Root className="select-none">
           <Breadcrumb.List>
-            <nav className="flex items-start gap-1">
-              {matches.map((match, index) => {
-                const { breadcrumb } = (match.handle || {}) as CustomHandle
-                const isFirst = index === 1
-                const isLast = index === matches.length - 1
+            {matches.map((match, index) => {
+              const { breadcrumb } = (match.handle || {}) as CustomHandle
+              const isFirst = index === 1
+              const isLast = index === matches.length - 1
 
-                if (!breadcrumb) return null
+              if (!breadcrumb) return null
 
-                return (
-                  <div key={match.pathname} className="flex items-center">
-                    {!isFirst ? <Breadcrumb.Separator className="mr-1" /> : null}
-                    <Breadcrumb.Item>
-                      <Breadcrumb.Item>
-                        {isLast ? (
-                          <Breadcrumb.Page>{breadcrumb(match.params)}</Breadcrumb.Page>
-                        ) : (
-                          <Breadcrumb.Link asChild>
-                            <Link to={match.pathname}>{breadcrumb(match.params)}</Link>
-                          </Breadcrumb.Link>
-                        )}
-                      </Breadcrumb.Item>
-                    </Breadcrumb.Item>
-                  </div>
-                )
-              })}
-            </nav>
+              return (
+                <Breadcrumb.Item key={index}>
```

Comment:
@vardanbansal-harness can we have `match.pathname` as key?

Reply:
Done.